azure-pipelines.yml

```diff
@@ -112,10 +106,10 @@ jobs:
       choco install -y mingw --force --version=6.4.0
     displayName: 'Install 64-bit mingw for 64-bit builds'
     condition: eq(variables['BITS'], 64)
-  - script: python -m pip install numpy cython==0.29.13 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath
+  - script: python -m pip install numpy cython==0.29.14 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath
     displayName: 'Install dependencies'
   - powershell: |
-      If ($(PYTHON_VERSION) -ne '3.5') {
+      If ($(PYTHON_VERSION) -ne '3.6') {
```

Comment:
With 3.5 out of the matrix we shouldn't need this conditional, but I'll just clean that up once the final matplotlib 3.2.0 comes out, presumably after the winter break.